tests/Doctrine/Tests/ORM/Mapping/PHPMappingDriverTest.php

```diff
@@ -33,6 +33,7 @@ protected function _loadDriver()
     public function testinvalidEntityOrMappedSuperClassShouldMentionParentClasses()
     {
         $this->createClassMetadata(DDC889Class::class);
+        $this->addToAssertionCount(1);
```

Comment:
The assertion is: no exceptions are thrown.
I did that based on [this search](https://github.com/doctrine/doctrine2/search?utf8=✓&q=DDC-889&type=Issues).